docs/local-development.md

```diff
@@ -0,0 +1,81 @@
+## Development with minikube and skaffold
+
+The network operator includes tooling to develop locally using `minikube` and `skaffold`.
+
+For more information on `skaffold` [see the official docs](https://skaffold.dev/).
+For more information on `minkube` [see the official docs](https://minikube.sigs.k8s.io/docs/).
+
+This development environment is designed to work on Linux amd64 and MacOS on Mac Silicon. It should be extensible for other platforms with some changes to the `minikube` installation.
+
+### Prerequisites:
+- `kubectl` CLI
+- `docker` CLI - but not necessarily the `docker` engine. This dev setup configures the CLI to use `minikube` as its `docker` engine.
+- `docker buildx` CLI plugin. [Installation instructions](https://github.com/docker/buildx?tab=readme-ov-file#manual-download)
+- A virtualization driver for minikube. This will be `qemu` for MacOS and `kvm2` for Linux. If this is not installed `minikube` will fail with an error pointing to install instructions.
+
+The `make` targets and scripts will install `minikube` and `skaffold` when running targets.
+
+### Local development with `skaffold`
+
+To spin up a cluster and deploy the network operator run:
+
+`make dev-skaffold`
+
+This will install `skaffold` and `minikube` if needed, spin up a `minikube` cluster called `net-op-dev`, and deploy the network operator in debug mode on the cluster. If there is an error ensure the [prerequisites](#prerequisites), including the virtualization driver, are met.
+
+Other useful `make` targets include `make dev-minikube` to setup the development cluster without skaffold and `make clean-minikube` to tear down the development cluster.
+
```

Comment:
can you add a note for those who have their home directory in a network drive that kvm driver may not work, failing with permissions issues.

it is recommended to use local home directory for minikube by specifyng `MINIKUBE_HOME=/local/path` either by exporting env var or shell var to make command.

Reply:
Thanks for figuring out the issue - I've added a note to point people in the right direction.